### DISCUSSION TITLE
File search always shows up empty

### DISCUSSION THREAD

**Discussion by FraserEmbrey** [PatrickF1/fzf.fish#93] [Q&A] [answered]
I have installed fzf, fd and bash. I also have ripgrep installed and I had the jethrokuan plugin but uninstalled with fisher rm before installing this one.

The commands all work except for the search files which always shows up as 0/0. The interface loads but it doesn't have any files to show.

I am using pop!_OS 20.04 and alacritty as my terminal emulator

**Comment by FraserEmbrey** [comment]
Realised fd needs to be aliased to fdfind. Might be worth noting that it only works with fdfind as linked, or making uses it uses the correct fd

**Comment by PatrickF1** [comment]
Hi Fraser! I'm confused. I don't have `fd` aliased to `fdfind`. How did you realize you needed to alias it to `fdfind`?

**Comment by FraserEmbrey** [comment, marked as answer]
> Hi Fraser! I'm confused. I don't have `fd` aliased to `fdfind`. How did you realize you needed to alias it to `fdfind`?

I went to the [fd github page](https://github.com/sharkdp/fd) linked to in your README and read the installation instructions more carefully. The Ubuntu section has a paragraph below the apt command saying:

> Note that the binary is called fdfind as the binary name fd is already used by another package. It is recommended that you add an alias fd=fdfind to your shells initialization file, in order to use fd in the same way as in this documentation.

**Comment by PatrickF1** [comment]
Thanks! I may create an FAQ and add it there.

**Comment by andy31415** [comment]
If anyone else stumbles upon this: in my case the fd binary was old and was not supporting the `--strip-cwd-prefix` argument. I debugged it by echo-ing the command used in `/.config/fish/functions/_fzf_search_directory.fish`

Fix ended up being `cargo install fd-find`

**Reply by manan-gup** [reply]
Had the same problem on Fedora. RPM has `fd` v8.2.1.
